### PR TITLE
Benchmark Utility Pallet

### PR DESCRIPTION
**Pull request by shawntabrizi**
This benchmarks all the extrinsics for the Utility Pallet.

Extrinsics related to multi-signing have multiple completion paths, so we benchmark them individually.

- batch
- as_sub
- as_multi
    - as_multi_create
    - as_multi_approve
    - as_multi_complete
- approve_as_multi
    - approve_as_multi_create
    - approve_as_multi_approve
- cancel_as_multi

### CODE REVIEW

File: frame/benchmark/src/lib.rs [hunk 1]
```diff
@@ -26,8 +26,7 @@ use frame_system::{self as system, ensure_signed};
 use codec::{Encode, Decode};
 use sp_std::prelude::Vec;
 
-#[cfg(feature = "runtime-benchmarks")]
-pub mod benchmarking;
+mod benchmarking;
```
Comment (marcio-diaz) [inline frame/benchmark/src/lib.rs]: :+1:

File: frame/treasury/src/benchmarking.rs [hunk 2]
```diff
@@ -16,6 +16,8 @@
 
 //! Treasury pallet benchmarking.
 
+#![cfg(feature = "runtime-benchmarks")]
```
Comment (bkchr) [inline frame/treasury/src/benchmarking.rs]: Why these changes?

Comment (shawntabrizi) [reply frame/treasury/src/benchmarking.rs]: imo it is cleaner. do you have an opinion with it?

Comment (bkchr) [reply frame/treasury/src/benchmarking.rs]: I find it the other way cleaner xD But it is okay.

**Comment by gnunicorn** [comment]
sorry, looks like it broke again @shawntabrizi  ...

**Comment by shawntabrizi** [comment]
@gnunicorn this is ready. the failing test on polkadot is another PR's issue
